Install.md

```diff
@@ -22,13 +22,8 @@ or for editable installs into the user directory:
 
     pip install --user -e .
 
-
-Installation of PorePy itself should be straightforward, following the instructions in Readme.md.
-
-To get the code fully working requires a few more steps, as described below.
-
 ## Installation on Windows
-Installation on Windows is currently (Spring 2021) rather easy, following the above instructions. The dependencies should be installed  using either `conda` or the Intel python distribution and then `pip install porepy` from source. 
+Installation on Windows is currently (Spring 2021) rather easy, following the above instructions. The dependencies should be installed using either `conda` or the Intel python distribution and then `pip install porepy` from source. 
```

Comment:
Should we update from "Spring 2021"?

Reply:
If we have verified more recently that this still works, I would say. What do you think?

Reply:
I haven't done it in a while

Reply:
Me neither (not since spring 2021). Should we leave it then, but perhaps change to 'at the time of writing'

Reply:
Though that change by itself would falsify the modified statement

Reply:
Leave as is for now and perhaps check at some point and update